src/NuGet.Clients/NuGet.Tools/NuGetFeedbackDiagnosticFileProvider.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.IO;
+using Microsoft.Internal.VisualStudio.Shell.Embeddable.Feedback;
+using NuGet.Common;
+
+namespace NuGetVSExtension
+{
+    [Export(typeof(IFeedbackDiagnosticFileProvider))]
+    public class NuGetFeedbackDiagnosticFileProvider : IFeedbackDiagnosticFileProvider
+    {
+        public IReadOnlyCollection<string> GetFiles()
+        {
+            // todo: https://github.com/NuGet/Home/issues/8605
+
+            return new List<string>()
+            {
+                Path.Combine(
```

Comment:
Path.Combine() returns full path as a string. Is it because of interface implementation we are forced to return a list instead of string itself ?

Reply:
The interface allows multiple files to be attached to feedback, but we only want to attach a single file.